lite/examples/style_transfer/ios/EXPLORE_THE_CODE.md

```diff
@@ -0,0 +1,215 @@
+  guard let mutableRawPointer = CVPixelBufferGetBaseAddress(buffer) else {
+    return nil
+  }
+  let count = CVPixelBufferGetDataSize(buffer)
+  let bufferData = Data(bytesNoCopy: mutableRawPointer, count: count, deallocator: .none)
+  var rgbBytes = [UInt8](repeating: 0, count: byteCount)
+  var index = 0
+  for component in bufferData.enumerated() {
+    let offset = component.offset
+    let isAlphaComponent = (offset % alphaComponent.baseOffset) == alphaComponent.moduloRemainder
+    guard !isAlphaComponent else { continue }
+    rgbBytes[index] = component.element
+    index += 1
+  }
+  if isModelQuantized { return Data(bytes: rgbBytes) }
+  return Data(copyingBufferOf: rgbBytes.map { Float($0) / 255.0 })
+}
+```
+
+#### Run inference
+
+Here's the code for getting the `RGB` data representation of the pixel buffer,
+copying that data to the input
+[`Tensor`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/swift/Sources/Tensor.swift),
+and running inference by invoking the `Interpreter`:
+
+```swift
+let outputTensor: Tensor
+do {
+  // Allocate memory for the model's input `Tensor`s.
+  try interpreter.allocateTensors()
+  let inputTensor = try interpreter.input(at: 0)
+
+  // Remove the alpha component from the image buffer to get the RGB data.
+  guard let rgbData = rgbDataFromBuffer(
+    thumbnailPixelBuffer,
+    byteCount: batchSize * inputWidth * inputHeight * inputChannels,
+    isModelQuantized: inputTensor.dataType == .uInt8
+  ) else {
+    print("Failed to convert the image buffer to RGB data.")
+    return
+  }
+
+  // Copy the RGB data to the input `Tensor`.
+  try interpreter.copy(rgbData, toInputAt: 0)
+
+  // Run inference by invoking the `Interpreter`.
+  try interpreter.invoke()
+
+  // Get the output `Tensor` to process the inference results.
+  outputTensor = try interpreter.output(at: 0)
+} catch let error {
+  print("Failed to invoke the interpreter with error: \(error.localizedDescription)")
+  return
+}
+```
+
+#### Process results
+
+If the model is quantized, the output `Tensor` contains one `UInt8` value per
+class label. Dequantize the results so the values are floats, ranging from 0.0
+to 1.0, where each value represents the confidence that a label is present in
+the image:
+
+```swift
+guard let quantization = outputTensor.quantizationParameters else {
+  print("No results returned because the quantization values for the output tensor are nil.")
+  return
+}
+
+// Get the quantized results from the output tensor's `data` property.
+let quantizedResults = [UInt8](outputTensor.data)
+
+// Dequantize the results using the quantization values.
+let results = quantizedResults.map {
+  quantization.scale * Float(Int($0) - quantization.zeroPoint)
+}
+```
+
+Next, the results are sorted to get the top `N` results (where `N` is
+`resultCount`):
+
+```swift
+// Create a zipped array of tuples [(labelIndex: Int, confidence: Float)].
+let zippedResults = zip(labels.indices, results)
```

Comment:
TODO: I should probably update this. This was copied from ImageClassifier and is not applicable to Style Transfer